packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-exhibit/illustrations-exhibit-filter/illustrations-exhibit-filter.component.html

```diff
@@ -1,30 +1,57 @@
-<div class="e-form-field">
-  <div class="e-input">
-    <i class="e-icon e-icon--search-color" aria-hidden="true"> </i>
-    <input
-      [value]="searchValue()"
-      (keyup)="handleSearchChange($event)"
-      aria-label="Search"
-      type="search"
-      [placeholder]="
-        locale() === 'en-GB' ? 'Search through all illustrations' : 'Søk gjennom alle illustrasjoner'
-      "
+<div class="illustration-exhibit-filter">
+  <div class="e-form-field">
+    <div class="e-input">
+      <i class="e-icon e-icon--search-color" aria-hidden="true"> </i>
+      <input
+        [value]="searchValue()"
+        (keyup)="handleSearchChange($event)"
+        aria-label="Search"
+        type="search"
+        [placeholder]="
+          locale() === 'en-GB' ? 'Search through all illustrations' : 'Søk gjennom alle illustrasjoner'
+        "
+      />
+      <button
+        class="e-btn e-btn--icon"
+        [ngClass]="{ 'e-invisible': !(searchValue() ?? '').length }"
+        (click)="handleClearSearch()"
+        aria-label="Clear search field"
+      >
+        <span class="e-btn__icon">
+          <i class="e-icon e-icon--close" aria-hidden="true"> </i>
+        </span>
+      </button>
+    </div>
+  </div>
+  <div class="local-theme-switch-toggles e-none-sm">
+    <elvia-radio-filter
+      name="Illustration color"
+      [items]="colorFilters"
+      [value]="colorFilterValue()"
+      (valueOnChange)="handleColorFilterChange($any($event))"
     />
-    <button
-      class="e-btn e-btn--icon"
-      [ngClass]="{ 'e-invisible': !(searchValue() ?? '').length }"
-      (click)="handleClearSearch()"
-      aria-label="Clear search field"
-    >
+    <app-local-theme-switch [theme]="theme()" (themeChange)="handleNewTheme($event)" />
+  </div>
+  <elvia-popover *ifViewportSize="['sm']">
+    <div slot="content" class="local-theme-switch-toggles-mobile">
+      <div>
+        <span class="e-text-md-strong">Color</span>
+        <elvia-radio-filter
+          name="Illustration color"
+          [items]="colorFilters"
+          [value]="colorFilterValue()"
+          (valueOnChange)="handleColorFilterChange($any($event))"
+        />
```

Comment:
This one should have a `[inlineStyle]="{ flexWrap: 'noWrap' }"` like the others as it makes the popover very narrow

Reply:
Ill do that, and then I can actually re-add the view encapsulation here 💡